backend/app.py

```diff
@@ -1,31 +1,30 @@
 import os
+from keras import backend as k
+import tensorflow as tf
 from tensorflow.keras.models import load_model
 from tensorflow.keras.preprocessing import image as img
 from tensorflow.keras.utils import img_to_array
-from keras import backend as k
+from tensorflow.keras.applications.resnet50 import ResNet50, preprocess_input
```

Comment:
```suggestion
from tensorflow.keras.applications.resnet50 import preprocess_input
```
resNet50 brauchen wir nicht mehr. Bitte rausnehmen :)